readthedocs/templates/core/project_detail_right.html

```diff
@@ -61,16 +61,25 @@ <h3>{% trans "Badge" %}</h3>
 
 
 {% block tags %}
-  {% if project.tags.count %}
     <h3>{% trans "Tags" %}</h3>
     <p>
-      {% for tag in project.tags.all %}
-        <a href="{% url "projects_tag_detail" tag.slug %}">{{ tag.name }}</a>{% if forloop.last %}{% else %}, {% endif %}
-      {% empty %}
-        <span class="quiet">{% trans "No tags" %}</span>
-      {% endfor %}
+
+    {% if project.tags.count %}
+        {% for tag in project.tags.all %}
+          <a href="{% url "projects_tag_detail" tag.slug %}">{{ tag.name }}</a>{% if forloop.last %}{% else %}, {% endif %}
+        {% endfor %}
+    {% else %}
+      {% trans "Project has no tags." %}
+      {% if request.user|is_admin:project %}
+
+        {% url 'projects_edit' project.slug as edit_url %}
```

Comment:
I don't remember this exactly but, can we just use the `url` tag like this to create a local variable? don't we need the `with` tag for this?

Reply:
Yep, it works!

Reply:
Docs here: https://docs.djangoproject.com/en/1.9/topics/i18n/translation/#blocktrans-template-tag